gradle/versions.gradle

```diff
@@ -4,7 +4,7 @@ ext {
   guavaVersion = '23.0'
   hadoopVersion = '2.7.4'
   hbaseVersion = '1.3.1'
-  kafkaVersion = '0.10.0.1'
+  kafkaVersion = '0.11.0.0'
```

Comment:
We should be really careful about this, as kafka module does not have any test cases ... (we should completely rewrite it in order to take full advantage of new kafka-client apis)